src/pages/test-results-search/components/search-result/__tests__/search-result.spec.ts

```diff
@@ -36,4 +37,48 @@ describe('SearchResultComponent', () => {
     });
   });
 
+  describe('driverRider', () => {
+    it('should return driver when cat is not A', () => {
+
+      const searchResult = {
+        costCode: null,
+        testDate: null,
+        driverNumber: null,
+        candidateName: null,
+        applicationReference: null,
+        category: TestCategory.B,
+        activityCode: null,
+      };
+      expect(component.driverRider(searchResult)).toEqual('Driver');
+    });
+
+    it('should return driver when cat is EUA1M1', () => {
+
+      const searchResult = {
+        costCode: null,
+        testDate: null,
+        driverNumber: null,
+        candidateName: null,
+        applicationReference: null,
+        category: TestCategory.EUA1M1,
+        activityCode: null,
+      };
+      expect(component.driverRider(searchResult)).toEqual('Rider');
+    });
+
+    it('should return driver when cat is EUAM2', () => {
+
+      const searchResult = {
+        costCode: null,
+        testDate: null,
+        driverNumber: null,
+        candidateName: null,
+        applicationReference: null,
+        category: TestCategory.EUAM2,
+        activityCode: null,
+      };
+      expect(component.driverRider(searchResult)).toEqual('Rider');
+    });
+  });
+
```

Comment:
this search result constant could get moved to a higher level so that its only instantiated once, then you could just update the searchResult.category at the start of each it function, would make for a cleaner commit